jekyll/_cci1/build-image-macos.md

```diff
@@ -49,7 +49,7 @@ The currently available Xcode versions are:
 
 We maintain a manifest of the software installed on our OSX and macOS build images. This includes version of the operating system, Xcode, Python, Ruby, etc.
 
-* [Xcode 9.2.0 (macOS 10.12 Sierra)](https://circle-macos-docs.s3.amazonaws.com/image-manifest/build-298/index.html).
+* [Xcode version 9.2.0 (macOS 10.12 Sierra)](https://circle-macos-docs.s3.amazonaws.com/image-manifest/build-298/index.html).
```

Comment:
(just adding the word `version` here)

Reply:
+1 for consistency within this section, but why aren't we using the word "version" earlier in the doc, too?

Reply:
🤷‍♂️ no reason - this document evolved to be this way really.